documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -57,21 +55,24 @@
    - Link to PR:
    - There likely aren't any changes at this point, but this can be opened with a small whitespace change so the PR is open and we can more easily hold the standard of making improvements incrementally since improvements are usually better done by collecting changes/notes along the way rather than just thinking about it at the end.
    - This will get merged in a `Post Release` step.
-- [ ] (patch release) Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from the last stable `release/vX.Y.Z` or `release/miner/vX.Y.Z` and make any further release-related changes to this branch.
+- [ ] (patch release) Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from the last stable `release/vX.Y.x` or `release/miner/vX.Y.x` and make any further release-related changes to this branch.
 - [ ] (minor release) Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from `master` and make any further release-related changes to this branch.
-- (minor release) `master` branch Version string updates
+- `master` branch Version string updates
+   - Skip this set of steps if you are patching a previos minor release. 
```

Comment:
I believe we still need to do these Version strings for path and minor release, unless it's a patch for a previous minor version.

Reply:
Yes, that is correct. It also reminds me that we should bump the version in master now